### PR TITLE
[Merged by Bors] - chore: split Data.Complex.Basic

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Complex/Basic.lean [hunk 1]
```diff
@@ -670,11 +668,11 @@ theorem normSq_nonneg (z : ℂ) : 0 ≤ normSq z :=
...
-@[simp]
-theorem range_normSq : range normSq = Ici 0 :=
-  Subset.antisymm (range_subset_iff.2 normSq_nonneg) fun x hx =>
-    ⟨Real.sqrt x, by rw [normSq_ofReal, Real.mul_self_sqrt hx]⟩
-#align complex.range_norm_sq Complex.range_normSq
+-- @[simp]
+-- theorem range_normSq : range normSq = Ici 0 :=
+--   Subset.antisymm (range_subset_iff.2 normSq_nonneg) fun x hx =>
+--     ⟨Real.sqrt x, by rw [normSq_ofReal, Real.mul_self_sqrt hx]⟩
+-- #align complex.range_norm_sq Complex.range_normSq
```
Comment (sgouezel) [inline Mathlib/Data/Complex/Basic.lean:675]: It looks like this one has disappeared from the split?

File: Mathlib/Data/Complex/Basic.lean [hunk 2]
```diff
@@ -970,353 +959,7 @@ theorem im_eq_sub_conj (z : ℂ) : (z.im : ℂ) = (z - conj z) / (2 * I) := by
...
-
-@[simp]
-theorem im_sum (f : α → ℂ) : (∑ i in s, f i).im = ∑ i in s, (f i).im :=
-  imAddGroupHom.map_sum f s
-#align complex.im_sum Complex.im_sum
-
 end Complex
+
+assert_not_exists Multiset
+assert_not_exists Algebra
```
Comment (eric-wieser) [inline Mathlib/Data/Complex/Basic.lean:965]: In the long run I'd be tempted to move the `Module` and `Algebra` structure into `Basic`; it's pretty annoying to find that structure missing (almost as bad as needing a separate import to make polynomials commutative...).
I think for that to be acceptable, we probably need to slim down `Algebra/Algebra/Basic.lean`

Either way, `assert_not_exists`s are harmless, and just force the next reviewer to actually think about what's going on :)

**Comment by semorrison** [comment]
bors merge

**Review by eric-wieser — COMMENTED** [review]
bors r-

This contains git submodules!

File: .lake/packages/Cli
Comment (eric-wieser) [inline .lake/packages/Cli:1]: Please remove these!

Comment (alreadydone) [reply .lake/packages/Cli:1]: https://github.blog/changelog/2021-01-11-delete-directory-from-the-web/ is how you delete a directory on GitHub.

File: Mathlib/Data/Complex/Basic.lean (hunk 1 above)
Comment (semorrison) [reply Mathlib/Data/Complex/Basic.lean]: Fixed.

File: .lake/packages/Cli
Comment (semorrison) [reply .lake/packages/Cli:1]: Ugh, sorry, @tydeu's changes to how lake stores its files are causing a mess during the transition.

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**